docs/features/template.md

```diff
@@ -0,0 +1,131 @@
+
+This section is important for producing high quality
+user-focused documentation. Goal here is to highlight **what this
+feature is all about**. This paragraph can be used to construct
+release notes when we do releases.
+
+Your Introduction should be one paragraph long. More detail
+should go into the following sections.
+
+## Motivation
+
+This section is for explicitly listing the motivation, goals and
+non-goals of this feature. Describe why the change is important and
+the benefits to users. Goal here is to highlight **why is this
+feature needed**. If there was an enhancement proposal written please
+link that in this section so that readers can go through the user
+stories there.
+
+### User-Stories/Use-Cases
+
+NOTE: If you had an enhancement proposal written for this feature
+then copy the user stories over from there OR provide a link
+to that enhancement proposal.
+
+(What new user-stories/use-cases does this feature introduce?)
+
+A user story should typically have a summary structured this way:
+
+1. **As a** [user concerned by the story]
+2. **I want** [goal of the story]
+3. **so that** [reason for the story]
+
+The “so that” part is optional if more details are provided in the description.
+A story can also be supplemented with examples, diagrams, or additional notes.
+
+e.g
+
+Story 1: Deny traffic at a cluster level
+
+As a cluster admin, I want to apply non-overridable deny rules to certain pod(s)
+and(or) Namespace(s) that isolate the selected resources from all other cluster
+internal traffic.
+
+For Example: The admin wishes to protect a sensitive namespace by applying an
+AdminNetworkPolicy which denies ingress from all other in-cluster resources
+for all ports and protocols.
+
+## How to enable this feature on an OVN Kubernetes cluster?
+
+Did you write new configs or knobs that need to be turned on
+for this feature? If so educate end users on those details.
+How can users disabled this feature if they don't want it?
+
+## Workflow Description
+
+Explain how the user will use the feature. Be detailed and explicit.
+Describe all of the actors, their roles, and the APIs or interfaces
+involved. Define a starting state and then list the steps that the
+user would need to go through to trigger the feature described in the
+enhancement. **A detailed architectural diagram is a must to showcase
+how this feature works in our CNI.** Remember that a picture can speak
+a thousand words.
+
+
+### Implementation Details
+
+Use the following sub-sections to explain more implementation details in a top down fashion
+
+### User facing API Changes
+
+If any API changes were done, they must be outlined here and
+look at the developer docs on how to generate API reference
+documentation for this feature. Provide a link to that API
+reference web page from this spot.
+Also add details on where the CRD lives; example is it in
+kubernetes or kubernetes-sigs or network-plumbing-working-group?
+
+### OVN Kubernetes Implementation Details
```

Comment:
Should it be `OVN-Kubernetes` instead?

Reply:
yes, using OVN-Kubernetes, will be better.

Reply:
hmm the recommendation I got from PR team/docs team when I wrote https://www.redhat.com/en/blog/how-the-new-ovn-kubernetes-architecture-helps-with-scaling-openshift was that we must call it either as `ovn-kubernetes` repo name OR `OVN Kubernetes`; hyphenating is not correct, we can make it `OVNKubernetes` as well ?

Reply:
going with OVN-Kubernetes, resolving issue